### PR TITLE
Update Issue Assignments and Include Obfuscated Requester Name in Issue Description

### LINKED ISSUE #2710: Include Obfuscated User Name in Body of DTS Service Requests
Add firstname & last initial to the body of new DTS service requests. See discussion in [this comment](https://github.com/cityofaustin/atd-data-tech/issues/2638#issuecomment-628222156).

### PR DESCRIPTION
**Pull request by johnclary**
Fixes [#2710](https://github.com/cityofaustin/atd-data-tech/issues/2710) and [#2636](https://github.com/cityofaustin/atd-data-tech/issues/2636).

Screen Shot 2020-05-20 at 6 08 12 PM

### CODE REVIEW

**Comment by johnclary** [comment]
Deployed this branch to prod. Will let it run for a few days.

File: _transforms.py [hunk 1]
```diff
@@ -27,3 +27,19 @@ def parse_attachment_url(obj):
...
     # format as markdown
     return f"[Attachment]({url}) ({size})"
+
+
+def parse_email(email_addr):
+    """
+    Extract `<First name> <Last intial>.` from email address
+    """
+    try:
+        first, last = email_addr.split("@")[0].split(".")
```
Comment (mddilley) [inline _transforms.py:37]: I'll have to remember to unpack variables like this in the future. 😎

**Review by mddilley — APPROVED** [review]
Looks great, John! Thanks for teaching me a new Python trick!

File: _transforms.py (hunk 1 above)
Comment (johnclary) [reply _transforms.py:37]: thanks @mddilley !

**Pull request merged**